app/src/main/java/waiter/Request.java

```diff
@@ -27,4 +27,12 @@ public String getUrl() {
     public String getProtocol() {
         return protocol;
     }
-}
+
+    public String getHeaders() {
+        return headers;
+    }
+
+    public String getBody() {
+        return body;
+    }
+}
```

Comment:
Nitpick: Newline here.

Reply:
Addressed in [this commit](https://github.com/jonmanahan/waiter/pull/24/commits/0294239e9ea74aa0369ba050ca9fa0ec8851d0fd)